fili-core/src/test/groovy/com/yahoo/bard/webservice/web/handlers/SplitQueryRequestHandlerSpec.groovy

```diff
@@ -67,6 +69,28 @@ class SplitQueryRequestHandlerSpec extends Specification {
         1         | MONTH     | month
         12        | MONTH     | year
     }
+    
+    def "Handler skips splitting for all time grain"() {
```

Comment:
Since we're using a `where` block, we should add the `@Unroll` annotation to the test so that each entry in the `where` block ends up being it's own test. The advantage of this is that we'll get info for each of the test failures, rather than just 1 entry for any of the cases failing.

As part of this, it would be good to add the interval attribute to the test name so that it shows up in the test report. You can do that by changing the name of the test to something like `Handler skips splitting for all time grain when the interval is #interval`